### PR TITLE
Education section added

### LINKED ISSUE #246: Eductation Section
### Summary
Add a section "Education" below the first section.

### What's new?
A new section that has owner's education background (college / university, degree, diploma, etc.)

### Motivation
Education background has always been a vital part of one's professional webpage. It gives the viewers of your website a better undertstanding of your capabilities and skills.

### PR DESCRIPTION
**Pull request by Ergouzii**
As discussed in Issue #246 , I created this PR and the changes can be previewed [here](https://developer-folio-with-education.netlify.app/).

Resolves #246.

### CODE REVIEW

**Comment by kartikcho** [comment]
Hi, thanks for working on this!

I haven't taken a full look at the code yet. I think we can add some interactivity to the cards by changing the `border-bottom-color` to black or our 'brand' purple in the `.education-card` class. 
Maybe we can pair it with an expanding transition if possible or any other minimal one to make the component more interesting.

Let me know what you think about adding this @Ergouzii!

**Comment by kartikcho** [comment]
I edited your original PR description to add a closure comment.
It will auto close the issue your PR is addressing when it's merged :)

**Comment by Ergouzii** [comment]
@kartikcho Great points! Just changed the border's color and added a sliding effect. Please check out [the site](https://developer-folio-with-education.netlify.app/) and let me know :)

**Comment by kartikcho** [comment]
I'm sorry if I wasn't clear earlier, I meant we can add hover effects like changing border color or a transition on hover. The purple looks great, just transition from the earlier grey bottom border to purple bottom border on card hover. Everything else looks cool!

**Comment by Ergouzii** [comment]
@kartikcho Haha sorry for that, just updated the effect. The border now will change to purple and be extended on hover. Please check and let me know!

**Comment by kartikcho** [comment]
This looks really neat!
Everything is set for a review but I ran into some small responsiveness bugs.

The class `.education-card-right` has `max-width: 70%;` but you've not set a media query to expand it to 100% for smaller screens which cause this issue,

image

Same for the bottom border, should set it 100% at the same width query at which it gets center aligned.

**Review by thevirtualbuddy — APPROVED** [review]

**Comment by Ergouzii** [comment]
@kartikcho Good catch! Just fixed it.

**Review by kartikcho — CHANGES_REQUESTED** [review]
Great PR! Just a few nits to fix and we can get this merged. Awesome work!

File: src/containers/education/Education.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.education-heading {
+  font-size: 56px;
```
Comment (Ergouzii) [inline src/containers/education/Education.css:2]: Sorry I misclicked "Resolve Conversation" on your comment for this code block. Yes I set it to 56px to be consistent with other sections. For example, `Skills.css` has a heading of 56px on large screens, 40px on medium screens, and 30px on small screens. I made this education heading the same way.

Comment (Ergouzii) [reply src/containers/education/Education.css:2]: @kartikcho

File: src/portfolio.js [hunk 2]
```diff
@@ -97,6 +97,36 @@ https://fontawesome.com/icons?d=gallery */
...
+// Your education background
+
+const educationInfo = {
+  viewEducation: true, // Set it to true to see education section
+  title: "Education",
+  schools: [
+    {
+      schoolName: "Harvard University",
+      logo: require("./assets/images/harvardLogo.png"),
+      degree: "Master of Science in Computer Science",
```
[the remark this answers is not in the document]
Comment (Ergouzii) [reply src/portfolio.js]: Changed to `subHeader`

File: src/portfolio.js [hunk 3]
```diff
@@ -97,6 +97,36 @@ https://fontawesome.com/icons?d=gallery */
   ]
 };
 
+// Your education background
+
+const educationInfo = {
+  viewEducation: true, // Set it to true to see education section
+  title: "Education",
```
[the remark this answers is not in the document]
Comment (Ergouzii) [reply src/portfolio.js]: Deleted `title` here

File: src/containers/education/Education.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './Education.css';
+import EducationCard from '../../components/educationCard/EducationCard';
+import { educationInfo } from '../../portfolio';
+
+export default function Education() {
+  return (
+    <div className="education-main-div" id="education">
+      <div>
+        <h1 className="education-heading">{educationInfo.title}</h1>
```
[the remark this answers is not in the document]
Comment (Ergouzii) [reply src/containers/education/Education.js]: Done

File: src/containers/education/Education.css [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  font-weight: 400;
+}
+
+.education-main-div {
```
[the remark this answers is not in the document]
Comment (Ergouzii) [reply src/containers/education/Education.css]: Done

File: src/containers/education/Education.css [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  margin: 0px auto;
+  margin-top: 2rem;
+}
+
+.education-main-div > * {
+  flex: 1;
+  margin-bottom: 30px;
+}
+
+.education-card-div {
```
[the remark this answers is not in the document]
Comment (Ergouzii) [reply src/containers/education/Education.css]: `education-card-div` is the container for all cards, and the name `education-card` is already been used by each individual card. Changed it to `education-cards`

File: src/components/header/Header.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import Headroom from "react-headroom";
 import "./Header.css";
-import {greeting, workExperiences} from "../../portfolio";
+import {greeting, educationInfo, workExperiences} from "../../portfolio";
 
 function Header() {
   const exp = workExperiences.viewExperiences;
```
[the remark this answers is not in the document]
Comment (Ergouzii) [reply src/components/header/Header.js]: Done

File: src/components/educationCard/EducationCard.css [hunk 8]
```diff
@@ -0,0 +1,140 @@
+.education-card {
+  display: flex;
+  flex-direction: row;
+}
+
+.education-card:hover + .education-card-border {
+  border-color: #a9a7f9;
+  width: 100%;
+  -webkit-transition: width 1s ease-in-out;
```
[the remark this answers is not in the document]
Comment (Ergouzii) [reply src/components/educationCard/EducationCard.css]: Changed to 0.6s

File: src/components/educationCard/EducationCard.css [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+  height: 6rem;
+  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.3);
+  border-radius: 50%;
+}
+
+.education-card-right {
+  max-width: 70%;
+}
+
+.education-text-degree {
```
[the remark this answers is not in the document]
Comment (Ergouzii) [reply src/components/educationCard/EducationCard.css]: Changed to `subHeader`

File: src/containers/education/Education.css (hunk 1 above)
Comment (kartikcho) [reply src/containers/education/Education.css:2]: Ah I see, it's alright then. 
Also, you can unresolve a comment if you accidentally clicked on it so don't worry about it. As a general good practice you should mark the comments addressed in a follow up commit as resolved.

File: src/components/educationCard/EducationCard.css [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+  -o-transition: width 0.6s ease-in-out;
+  transition: width 0.6s ease-in-out;
+}
+
+.education-head {
+  display: flex;
+  flex-direction: row;
+}
+
+.education-div-school {
```
Comment (kartikcho) [inline src/components/educationCard/EducationCard.css:31]: Avoid using element names in classes like you've done here. Instead go for smaller descriptive names like `education-school-container` or something

File: src/containers/education/Education.css [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+  margin: 0px auto;
+  margin-top: 2rem;
+}
+
+.education-section > * {
+  flex: 1;
+  margin-bottom: 30px;
+}
+
+.education-cards {
```
Comment (kartikcho) [inline src/containers/education/Education.css:19]: My bad earlier, to make it a bit different, we can go for `education-card-container` or something of the sort. What do you think?

File: src/containers/education/Education.js [hunk 12]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './Education.css';
+import EducationCard from '../../components/educationCard/EducationCard';
+import { educationInfo } from '../../portfolio';
+
+export default function Education() {
+  return (
+    <div className="education-section" id="education">
+      <div>
```
Comment (kartikcho) [inline src/containers/education/Education.js:9]: Any specific reason for this empty div?

**Review by kartikcho — APPROVED** [review]
Just one last round of changes left to address and we're good!

**Comment by Ergouzii** [comment]
@kartikcho Done the last round of fixes :)

**Review by kartikcho — APPROVED** [review]
Everything looks great, LGTM!

**Comment by kartikcho** [comment]
@all-contributors add @Ergouzii for code and design.

**Pull request merged**